components/icon/src/AccessibilityIcon.tsx

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import { Icon, IconProps } from './Icon';
+
+export const AccessibilityIcon = ({
+	assistiveText = 'Accessibility',
+	copyrightYear = '2020',
+	size = 'medium',
+	overrides,
+	...props
+}: Omit<IconProps, 'icon'>) => (
```

Comment:
Where is the 'icon' type encountered? If it is encountered somewhere, is that the reason for using the Omit constructor?

Reply:
It means we are using IconProps type but not using icon from IconProps. 
All the icons are from baseIcon. as you can see below, baseIcon component is using icon prop. 
Omit has been used to prevent duplication